readthedocs/templates/profiles/private/security_log.html

```diff
@@ -0,0 +1,63 @@
+{% extends "profiles/base_profile_edit.html" %}
+
+{% load i18n %}
+{% load pagination_tags %}
+
+{% block title %}{% trans "Security Log" %}{% endblock %}
+
+{% block profile-admin-security-log %}active{% endblock %}
+
+{% block edit_content_header %} {% trans "Security Log" %} {% endblock %}
+
+{% block edit_content %}
+
+<p>
+  {% blocktrans trimmed with days_limit as days_limit %}
+  Showing logs from the last {{ days_limit }} days.
+  {% endblocktrans %}
+</p>
+
+{% autopaginate object_list 15 %}
+
+<div class="module">
+  <div class="module-list">
+    <div class="module-list-wrapper">
+      <ul>
+        {% for log in object_list %}
+        <li class="module-item">
+          {% blocktrans trimmed with ip=log.ip browser=log.browser %}
+          Authenticated from
```

Comment:
The verb form here sounds like "You authenticated from 1.2.3.4" instead of "Someone was authenticated from 1.2.3.4"

```suggestion
          Authentication attempt from
```

However :point_up: suggests we are logging "attempts". If we want to log authentication success vs authentication failure, we'll want more specific language of course:

- `User meatloaf authenticated from 1.2.3.4`
- `User meatloaf failed to authentication from 1.2.3.4`
- `Authentication failure from 1.2.3.4`

Reply:
Is it okay to put the username on each entry? All these logs are for the current user.

Reply:
Ah I missed that this is for the individual profile. Yeah you're right then, the username is superfluous in this list.